week-5/B-callbacks/exercise.js

```diff
@@ -37,15 +43,22 @@ function exercise2(result) {
   EXERCISE 3
   =======
 
-  Write a function that calls the callback with the parameter "Hello from the
+  The exercise3 function will be called for you automatically. It is called
+  with a callback function as the first parameter. Call this callback function
+  (hint: use () parentheses) with the parameter "Hello from the callback" and
+  it will insert it into the #exercise3 element.
+
+  EXPECTED RESULT: The #exercise3 element has textContent = "Hello from the
   callback"
 
-  What happens to the #exercise3 element? Why does this happen? (Hint: look at
-  line 63 below)
+  Can you explain why we get this result? Write down your explanation in a
+  comment below (Hint: look at line 63 below)
```

Comment:
I think line 63 is already out of date! Maybe something like "Look at the comment inside the function"

Reply:
You're right that it was out of date, but actually I was referencing the `exercise3` callsite (now on line 77). Updated to not use a line number.